Gemfile

```diff
@@ -54,7 +54,6 @@ group :gcs do
 end
 
 group :gae do
-  gem 'rubyzip'
 end
```

Comment:
The library was used to unpack the downloaded file prior to this rewrite, so I think it's safe to assume it is not used anymore. Will do so.

Along this note, I added `require`s for `'rubygems/package'` and `'zlib'`. Should they be added here? I don't know anything about dependencies in Ruby.